daemon/daemon_test.go

```diff
@@ -248,6 +264,11 @@ func (s *daemonSuite) TestPolkitInteractivity(c *check.C) {
 	put.Header.Set(client.AllowInteractionHeader, "true")
 	c.Check(cmd.canAccess(put, nil), check.Equals, true)
 	c.Check(s.lastPolkitFlags, check.Equals, polkit.CheckAllowInteraction)
+
+	// bad values are logged and treated as false
```

Comment:
(nitpick) the comment is slightly misleading. AFAICT it does not test that the value is actually logged(?).